### PR TITLE
857 add mediaType codelist

### LINKED ISSUE #857: Add IANA Media Types codelist
`Document.format` refers to the codelist, but the schema doesn't specify it. We would need a script like the one for the currency codelist.

http://www.iana.org/assignments/media-types/

### PR DESCRIPTION
**Pull request by duncandewhurst**
Continued from #1308 

Closes #857

### CODE REVIEW

File: docs/history/changelog.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 * [#1225](https://github.com/open-contracting/standard/pull/1225) Add 'securityClearanceDeadline' code from the EU extension to the `milestoneType` codelist.
 * [#1238](https://github.com/open-contracting/standard/pull/1238) Open the `releaseTag` codelist.
 * [#1243](https://github.com/open-contracting/standard/pull/1243) Add the 'procurement' and 'contractClose' codes from the PPP extension to the `milestoneType` codelist.
+* [#1308](https://github.com/open-contracting/standard/pull/1308) Add the `mediaTypes` codelist.
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
* [#1308](https://github.com/open-contracting/standard/pull/1308) Add the `mediaType` codelist.
```

File: docs/schema/codelists.md [hunk 2]
```diff
@@ -152,6 +152,16 @@ The extended procurement category codelist is used to provide additional detail
 :file: ../../build/current_lang/codelists/extendedProcurementCategory.csv
 ```
 
+### Media type
+
+The media type codelist is based on the [IANA Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) with one additional code for offline documents: 'offline/print'. The codelist omits deprecated and obsoleted media types.
```
Comment (jpmckinney) [inline docs/schema/codelists.md]: Putting all differences with IANA in one sentence:

```suggestion
The media type codelist is based on the [IANA Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) list. The media type codelist adds an exceptional code for printed documents ('offline/print'), and omits any media type that is marked as deprecated or obsolete by IANA.
```

File: schema/release-schema.json [hunk 3]
```diff
@@ -1033,11 +1033,13 @@
         },
         "format": {
           "title": "Format",
-          "description": "The format of the document, using the open [IANA Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) codelist (see the values in the 'Template' column), or using the 'offline/print' code if the described document is published offline. For example, web pages have a format of 'text/html'.",
+          "description": "The format of the document, using the open [media type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#media-type) codelist. For example, web pages have a format of 'text/html'. Use the 'offline/print' code if the document is published offline.",
```
Comment (jpmckinney) [inline schema/release-schema.json]: Avoid imperative sentences that are instructions to publishers:

```suggestion
          "description": "The format of the document, using the open [media type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#media-type) codelist. For example, a web page has a format of 'text/html'. The 'offline/print' code is reserved for printed documents not available in other formats.",
```

**Comment by duncandewhurst** [comment]
There seems to be an issue with the deploy script:
```
Run curl -sS https://raw.githubusercontent.com/open-contracting/deploy/main/deploy-docs.sh | bash -
+ REF=857-IANA-codelist-2
+ COMPONENT=
+ '[' false == true ']'
+ PREFIX=staging/
+ SUFFIX=
+ curl --silent --connect-timeout 1 standard.open-contracting.org:8255
+ true
+ rsync -az --delete-after build/ ocds-docs@standard.open-contracting.org:web/staging/857-IANA-codelist-2
Warning: Permanently added the RSA host key for IP address '5.28.62.151' to the list of known hosts.
+ ocdsindex sphinx build/ https://standard.open-contracting.org/staging/857-IANA-codelist-2/
+ ocdsindex index https://standard.open-contracting.org:9200 documents.json
+ '[' false == true ']'
bash: line 35: RELEASE: unbound variable
Error: Process completed with exit code 1.
```

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
Merging as I'm working to put all scripts into manage.py, like in other specification repositories.

**Pull request merged**